markdown/ch3.markdown

```diff
@@ -1,8 +1,8 @@
 # Networks of Image Regions with ndimage
 
-> Tyger Tyger, burning bright,  
-> In the forests of the night;  
-> What immortal hand or eye,  
+> Tyger Tyger, burning bright,
+> In the forests of the night;
+> What immortal hand or eye,
```

Comment:
@stefanv can we try to do something about this in conversion, rather than ignoring it altogether? The linebreaks *have* to be there for correct formatting. Can you see [this comment](https://github.com/elegant-scipy/elegant-scipy/pull/251#discussion_r115136345)? Maybe it's just a matter of changing `<br/>` to `<br>` during the conversion? (Or replace them with `<br>` directly?)